cli/ballerina-cli/src/main/resources/cli-help/ballerina-openapi.help

```diff
@@ -82,6 +82,9 @@ OPTIONS
         --client-methods <resource|remote>
             This option can be used in client generation to select the client method type, which can be `resource` or `remote`. (The default option is `remote`).
 
+        --without-data-binding
+            This option can be used in service generation to generate a low level service without data binding.
```

Comment:
```suggestion
            This option can be used in service generation to generate a low-level service without any data binding logic.
```

Reply:
Should be as follows:

`This option can be used in the service generation to generate a low-level service without any data-binding logic.`

Reply:
Update. Thank you